README.md

```diff
@@ -1 +1,46 @@
-# ToDo-Review
\ No newline at end of file
+# Today's To Do
+
+## This is a simple daily todo application
+
+## Getting Started
+
+1. copy the Url from my github account
+2. clone the below url to your local machine
+3. Run git clone git@github.com:ismailakinkunmi/portfolio-site.git
+
```

Comment:
It is also the best practice to give clear procedures for using your project. You have done a great job listing the step by steps but I kindly recommend that it will be much better if you add some links and the terminal commands so that even the beginner can know what to do. 👍